src/components/HeyButton/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+
+export const HeyButton = () => {
+  const [count, setCount] = useState(0);
+  const handleClick = () => {
+    if (count < 19) {
+      setCount(count => count + 1);
+    } else {
+      alert('20へぇ到達しました！'); //上限回数アラート（確認用です）
+    }
```

Comment:
このままでも良いですが、以下のようにしてもよいですね〜
コードのネストはなるべく浅くしたいのと、処理しない場合は早めに return しちゃいたい、というやつですね

```suggestion
    if (count >= 20) {
      alert('上限は20へぇです！'); //上限回数アラート（確認用です）
      return;
    }
      setCount(count => count + 1);
```

Reply:
ありがとうございます！
うぁーめっちゃ勉強になります😭
おっしゃられている通りだと思います✨🙇‍♂️
速攻で修正いたします😊

Reply:
上記内容で修正させて頂きました✨🙏